js/components/graph/Sidebar.js

```diff
@@ -5,14 +5,25 @@ export default class Sidebar extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      collapsed: true
+      collapsed: true,
+      results: []
     };
   }
 
   toggleSidebar = () => {
       this.setState({ collapsed: !this.state.collapsed });
   }
 
+  filteredSearch = (Query) => {
```

Comment:
@EclipseIsDead Parameters should have lowercase names (`query`, not `Query`). Please change this one and `Course` below.